package/MDAnalysis/core/topologyobjects.py

```diff
@@ -116,13 +116,12 @@ def __hash__(self):
         return hash((self._u, tuple(self.indices)))
 
     def __repr__(self):
-        indices = (self.indices if self.indices[0] < self.indices[-1]
-                   else self.indices[::-1])
+        "Return representation in same order of AtomGroup indices"
```

Comment:
Could you add a `versionchanged` in at the top of this class to indicate that this change has happened?

Reply:
Updates made. Think they should be okay.